src/Repository/ApActivityRepository.php

```diff
@@ -95,6 +95,10 @@ public function findLocalByApId(string $apId): ?array
                         'type' => Post::class,
                     ];
                 } else {
+                    if (0 === $id) {
```

Comment:
Maybe also explain this code, just for future references so we still understand why this was added.

```suggestion
                    // Ignore processing $id of zero, because ...
                    if (0 === $id) {
```